draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -133,6 +135,31 @@ CLOSE_STREAM frames are ack-eliciting. When lost, they MUST be retransmitted,
 unless the stream state has transitioned to "Data Recvd" or "Reset Recvd" due
 to transmission and acknowledgement of other frames (see {{multiple-frames}}).
 
+## ENOUGH Frame
+
+Conceptually, the CLOSE_STREAM frame is a STOP_SENDING frame with an
+added Reliable Size field.
+
+ENOUGH Frame {
+  Type (i) = TBD,
+  Stream ID (i),
+  Application Protocol Error Code (i),
+  Reliable Size (i),
+}
+
+The ENOUGH frames contain the following fields:
+
+Stream ID:  A variable-length integer encoding of the stream ID of
+      the stream being terminated.
```

Comment:
```suggestion
      the stream being ignored.
```
Take it or leave it, as this is a picky suggestion. It might make sense to use the same phrase [as that of STOP_SENDING](https://quicwg.org/base-drafts/rfc9000.html#name-stop_sending-frames).